ui/app/routes/vault/cluster/settings/auth/configure/section.js

```diff
@@ -86,7 +86,7 @@ export default Route.extend(UnloadModelRoute, {
         // if you haven't saved a config, the API 404s, so create one here to edit and return it
         if (e.httpStatus === 404) {
           config = this.store.createRecord(modelType, {
-            id: backend.id,
```

Comment:
I can't find where, but _somewhere_ a record is being created by ember-data (that doesn't persist to the store) and so this was causing the following error
```
router.js:1129 Error while processing route: vault.cluster.settings.auth.configure.section 
The id oidcNEW has already been used with another 'auth-config/oidc' record.
 Error: The id oidcNEW has already been used with another 'auth-config/oidc' record.
```

Reply:
<img width="1089" alt="Screenshot 2024-10-15 at 2 24 28 PM" src="https://github.com/user-attachments/assets/1d9e3a10-f67e-4680-af10-2c7d2166d53f">